core/src/com/po/fuck/view/classdrawers/Misc.java

```diff
@@ -4,9 +4,33 @@
 import com.badlogic.gdx.graphics.g2d.Animation;
 import com.badlogic.gdx.graphics.g2d.TextureRegion;
 import com.badlogic.gdx.graphics.g2d.Sprite;
+import com.badlogic.gdx.utils.JsonValue;
+import com.po.fuck.Assets;
+import com.po.fuck.model.Core;
 
 public class Misc {
     public static Animation<TextureRegion> getAnimation(String internalPath){
```

Comment:
Probably can be merged with `getComplixAnimation`. And probably accept object name instead of the path.